### DISCUSSION TITLE
Allow escaping brackets in payloads

### DISCUSSION THREAD

**Discussion by vladimir-t** [directus/directus#13838] [Feature Requests]
### Describe the Bug

When i try to post json with curly braces, it try to render it and on my backend i get undefined instead of plain string.
It's cool we have variables and mustache functionality, but sometimes we need to escape it

### To Reproduce

Create flow with Webhook/Request URL
Set POST, enter your mock url
Set Header: Content-Type=application/json
Set payload:
```
{
 "foo":"{{bar}}"
}
```
Trigger flow

### Errors Shown

Data my backend receive:
```
{ foo: 'undefined' }
```
Expected:
```
{ foo: '{{bar}}' }
```

### What version of Directus are you using?

9.12.2

### What version of Node.js are you using?

16.13.2

### What database are you using?

Postgresql 12

### What browser are you using?

Chrome

### How are you deploying Directus?

running locally

**Comment by rijkvanzanten** [comment]
Started a discussion here to see if there's any special character combo we can rely on to escape certain bracket sets over on https://github.com/userpixel/micromustache/issues/109

**Comment by vladimir-t** [comment]
JFYI I'll explain what i actually try to achieve:
I made a data flow, where i create couple of collections (with relations) via directus rest api. Everything is ok, but relation field renders with mustache template:

```
  "meta": {
        ...
        "options": {
          "template": "{{name}}, {{status}}"
        }
        ...
}
```
And directus render it on ongoing request (or on flow save, i don't know)

As a result i get "undefined, undefined" at relation field

**Reply by rijkvanzanten** [reply]
Ah yeah that makes total sense, and indeed, we'd need some sort of way to escape those brackets 🤔 (cc @nickrum in case you have any cool ideas)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
